src/SharedUtilities/models.jl

```diff
@@ -22,6 +27,24 @@ An abstract type for all models.
 """
 abstract type AbstractModel{FT <: AbstractFloat} end
 
+"""
+    AbstractImExModel{FT} <: AbstractModel{FT}
+
+An abstract type for models which have both an implicit and an explicit tendency.
```

Comment:
"which have tendency terms which must be treated implicitly (and which may also have tendency terms that can be treated explicitly)"